specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1726 @@
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RequestState": {
+      "description": "Request status.",
+      "enum": [
+        "Accepted",
+        "Created",
+        "Invalid",
+        "Succeeded",
+        "Failed",
+        "InProgress",
+        "Canceled"
+      ],
+      "readOnly": true,
+      "type": "string",
+      "x-ms-enum": {
+        "name": "RequestState",
+        "modelAsString": true
+      }
+    },
+    "OperationsStatus": {
+      "description": "A long running process operation Status. Based on - https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The operation status identifier. It should match what is used to GET the operation resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "It must match the last segment of the id field, and will typically be a GUID / system generated value."
+        },
+        "type": {
+          "description": "The type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "$ref": "#/definitions/RequestState",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "AssignedToSubscription": {
+      "type": "object",
+      "description": "SubscriptionIds and quota allocated to subscriptions from the GroupQuota.",
+      "additionalProperties": {
+        "type": "integer",
+        "format": "int64",
+        "description": "Group Quota assigned to subscription."
+      }
+    },
+    "AssignedQuotaToSubscriptionList": {
+      "type": "array",
+      "readOnly": true,
+      "description": "Assigned Group Quota to subscriptions.",
+      "items": {
+        "$ref": "#/definitions/AssignedToSubscription"
+      },
+      "x-ms-identifiers": []
+    },
+    "GroupQuotaLimit": {
+      "type": "object",
+      "description": "Group Quota limit.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Group Quota properties for the specified resource.",
+          "$ref": "#/definitions/GroupQuotaDetails"
+        }
+      }
+    },
+    "GroupQuotaDetails": {
+      "type": "object",
+      "description": "Group Quota details.",
+      "properties": {
+        "limit": {
```

Comment:
>limit [](http://example.com/codeflow?start=9&length=5)

does it have min and max value?

Reply:
Also the value varies from RP to RP - Compute vs Network RP quota will be different.